### DISCUSSION TITLE
stuck eforms from october 24th

### DISCUSSION THREAD

**Discussion by OliverStingl** [OP-TED/eForms-SDK#1061] [Q&A] [answered]
we have some stuck eforms, notice uuid is 9da83454-8fb9-471d-915b-49b4c5f0ee14.
Versions 1 is published, version 2 is stopped, but versions 3 to 6 are stuck on status "SUBMITTED", is this an issue with the publication process or are those eforms invalid?

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this non-SDK related concern, please get in touch directly with the helpdesk.
This may also be a duplicate of the previous discussion: https://github.com/OP-TED/eForms-SDK/discussions/1053
KR
